class/defaults.yml

```
@@ -1,4 +1,71 @@
 parameters:
-  monitoring:
-    =_metadata: {}
-    namespace: syn-monitoring
+  syn_kube_prometheus:
+    # Dependency resolution
+    jsonnetfile_parameters:
+      kube_prometheus_version: ${syn_kube_prometheus:_kube_prometheus_version:${syn_kube_prometheus:kubernetes_version}}
+    kubernetes_version: 1.21
```

I'm pretty sure something somewhere would be very unhappy to see `kubernetes_version` as a float.

```suggestion
    kubernetes_version: '1.21'
```